### PR TITLE
Update to v1.9.1

### PR DESCRIPTION
**Pull request by setu4993**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by setu4993** [comment]
@conda-forge-admin, please rerender

**Comment by setu4993** [comment]
@mariusvniekerk, @kwilcox: Can you please take a look and provide feedback? Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,34 +7,41 @@ package:
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
-    build:
-        - python
-        - setuptools
-    run:
-        - python
+  build:
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: this section should be removed.  we only need host / run

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,34 +7,41 @@ package:
...
 test:
   imports:
     - avro
+    - avro.schema
 
 about:
   home: http://avro.apache.org
   license: Apache 2.0
+  license_family: APACHE
+  license_file:
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: the license file is mandatory for apache licenses.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -8,34 +7,41 @@ package:
...
+    - python
 
 test:
   imports:
     - avro
+    - avro.schema
 
 about:
   home: http://avro.apache.org
   license: Apache 2.0
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -8,34 +7,41 @@ package:
 source:  # [py27 or py3k]
   # Avro exist as two versions in apache avro for py2 and py3.
   fn: avro-{{ version }}.tar.gz  # [py27]
-  url: {{ mirror }}/avro/avro-{{ version }}/py/avro-{{ version }}.tar.gz  # [py27]
-  sha256: 'e4181a9dc686c82cbc9bb496787d659d55582c44ad74b69924c45f80f3d513a1'  # [py27]
+  url: https://files.pythonhosted.org/packages/a1/9a/a2c83f6105d975ec6680b2cab79b4e3bd51ae33f31c10debe8d2485c8f80/avro-{{ version }}.tar.gz  # [py27]
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: can you change the url used here to 
```
url: https://pypi.io/packages/source/{{ pip_name[0] }}/{{ pip_name }}/{{ pip_name }}-{{ version }}.tar.gz
```

and add a section near the setters at the top doing

```
{% set pipname = 'avro' %}  # [py <= 27]
{% set pipname = 'avro-python3' %}  # [py >= 34]
```

**Review by mariusvniekerk — CHANGES_REQUESTED** [review]
Thanks for all this hard work.  Just some minor changes to have this conform to our current practices.

File: recipe/meta.yaml (hunk 4 above)
Comment (setu4993) [reply recipe/meta.yaml]: Couldn't figure out what the URL structure for `pypi.io/packages` was. Very helpful, thanks!

File: recipe/meta.yaml (hunk 2 above)
Comment (setu4993) [reply recipe/meta.yaml]: Adding it now 👍.

File: recipe/meta.yaml (hunk 1 above)
Comment (setu4993) [reply recipe/meta.yaml]: I removed it, but I'm curious why it is not required. The example of the `meta.yaml` file [here](https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs.html#editing-the-meta-yaml-file) shows only build and run sections (no host), but the attached example has host and run.

File: recipe/meta.yaml (hunk 2 above)
Comment (setu4993) [reply recipe/meta.yaml]: Is there a better way to store (and reference) the license file? I don't think putting it in the `recipe` folder is a good answer. I tried [this](https://github.com/conda-forge/python-avro-feedstock/blob/fc2005303c4a2d18c3aee49df651ac3287e7ba90/recipe/meta.yaml#L39) but that fails, link didn't work either.

**Comment by setu4993** [comment]
Thank you for the review and feedback, @mariusvniekerk!

File: recipe/meta.yaml (hunk 2 above)
Comment (setu4993) [reply recipe/meta.yaml]: Never mind, figured out where the license was being stored in the package and referring to that instead.

**Comment by setu4993** [comment]
@mariusvniekerk : I have made the requested changes and squashed the commits. Can you please review it again and comment back if I missed something? Thanks!

**Pull request merged**